src/middleware/session/mod.rs

```diff
@@ -601,7 +633,11 @@ where
     /// # ;}
     /// ```
     pub fn allow_cross_site_usage(self) -> NewSessionMiddleware<B, T> {
-        self.remove_cookie_option(CookieOption::SameSite(SameSiteEnforcement::Lax))
+        let cookie_config = SessionCookieConfig {
+            same_site: SameSiteEnforcement::Lax,
+            ..(*self.cookie_config).clone()
+        };
+        self.rebuild_new_session_middleware(cookie_config)
```

Comment:
Rustdoc for this function:

> Removes the `SameSite` cookie attribute, allowing cross-site requests to include the cookie.

We currently have no mechanism to remove the `SameSite` attribute, so we'll need a `None` variant, or to wrap the field in an `Option<_>` to implement this correctly.